terraform/modules/private-dns-zone-extend/variables.tf

```diff
@@ -0,0 +1,12 @@
+variable "business_unit_name" {
```

Comment:
Is this variable type right? It looks like it's being treated as a string. Also, is it necessary?

Reply:
variable changed